locales-lt-LT.xml

```diff
@@ -258,13 +257,13 @@
 
     <!-- VERB ROLE FORMS -->
     <term name="container-author" form="verb"></term>
-    <term name="director" form="verb">directed by</term>
+    <term name="director" form="verb">vadovavo</term>
```

Comment:
You are right, previous I looked at 234 and 246 lines with same short forms.
Fixed in https://github.com/citation-style-language/locales/pull/154/commits/dfea4a9b0cb1eb136f251cd0f02cc07a628b7413